pivot/views/data_api.py

```diff
@@ -26,12 +30,19 @@ def _get_csv(self):
             response = urlopen(url)
             data = response.read()
         except ValueError:
```

Comment:
I couldn't think of a better way to catch an exception in this except branch... 

Nested excepts do look kinda icky but they should work